src/SessionSetup/ConnectToStadiaWidget.cpp

```diff
@@ -91,6 +94,14 @@ ConnectToStadiaWidget::ConnectToStadiaWidget(QWidget* parent)
                    [this]() { emit InstanceReloadRequested(); });
 
   SetupStateMachine();
+
+  if (absl::GetFlag(FLAGS_enable_project_selection)) {
+    ui_->instancesFilterLineEdit->hide();
+    ui_->refreshButton->hide();
```

Comment:
It took me a while to figure out that `refreshButton` is being replaced with `refreshButton_2` and that the one being hidden here is the old one. I guess this is an intermediate step so a TODO might help.

Reply:
I added a few comments -> done